crates/topos-tce-proxy/src/client.rs

```diff
@@ -381,6 +381,7 @@ impl TceClientBuilder {
                                 let tce_endpoint = tce_endpoint.clone();
                                 let tce_grpc_client = tce_grpc_client.clone();
                                 let context_backoff = context.clone();
+                                // TODO: Push certificates to the TCE one by one
```

Comment:
I don't understand this TODO. I think it would be better to create a proper ticket with a full description.